tests/petsc/petsc_snes_05.cc

```diff
@@ -63,6 +63,7 @@ main(int argc, char **argv)
     auto       commrank = Utilities::MPI::this_mpi_process(MPI_COMM_WORLD);
     VectorType x(MPI_COMM_WORLD, 1, commrank == commsize - 1 ? 1 : 0);
     x(0) = 1.0;
+    x.compress(VectorOperation::insert);
```

Comment:
Ah, would you mind submitting these patches to the test suite separately? They are logically independent and could be merged with no further discussion.